app/src/main/java/com/esri/arcgisruntime/opensourceapps/datacollection/views/fragments/DataCollectionFragment.kt

```diff
@@ -242,7 +255,8 @@ class DataCollectionFragment : Fragment() {
      */
     private fun resetIdentifyResult() {
         mapViewModel.identifiableLayer?.clearSelection()
-        identifyResultViewModel.resetIdentifyResult()
+        identifyResultViewModel.resetIdentifyLayerResult()
+        popupViewModel.clearPopup()
```

Comment:
This call can be done in `identifyResultViewModel.resetIdentifyLayerResult()`

Reply:
updated